spec/factories/qc_results.rb

```diff
@@ -4,7 +4,7 @@
   factory :qc_result do
     labware_barcode { 'AB1234' }
     sample_external_id { 'Samp-1234' }
-    qc_assay_type { create(:qc_assay_type) }
+    qc_assay_type
```

Comment:
does this automatically associate the factory or does this need` association :qc_assay_type`

Reply:
Ah just seen the rubocop examples. Would be nice to standardise all of them to be explicit but its not necessary, up to you.

Reply:
I just tried to make it [explicit](https://github.com/rubocop/rubocop-factory_bot/blob/afc310efc27661d673349aa560b3559dae1ab7b5/lib/rubocop/cop/factory_bot/factory_association_with_strategy.rb) (`association :qc_assay_type`), but rubocop actually reverted it back to implicit (`qc_assay_type`)